R/approach_gaussian.R

```diff
@@ -14,10 +14,14 @@
 setup_approach.gaussian <- function(internal,
                                     gaussian.mu = NULL,
                                     gaussian.cov_mat = NULL, ...) {
+
+
   parameters <- internal$parameters
   x_train <- internal$data$x_train
   feature_specs <- internal$objects$feature_specs
 
+  # TO DO: gaussian.mu should probably be extracted from internal$parameters$gaussian.mu...
```

Comment:
Yes, you are right. This is a bug. I suggest doing it through the insert_defaults function as for ctree. (For gaussian the defaults are NULL, so it is not strictly necessary, but consistency is nice). Should not fix that in this PR, though.

Reply:
I'll make another PR with this.